### PR TITLE
fix filetree fix function naming relocate tests some class renaming

### PR DESCRIPTION
**Pull request by Rorzakh-tix**

### CODE REVIEW

File: src/library/database/models/base_model.py
Comment (mikhaillazko) [inline src/library/database/models/base_model.py:1]: let's remove dir models and move file to parent dir - database

File: src/library/database/database_config.py [hunk 1]
```diff
@@ -25,12 +23,12 @@ def assemble_db_connection(cls, v: Optional[str], info: ValidationInfo) -> str:
         return f'postgresql+asyncpg://{user}:{password.get_secret_value()}@{server}/{db}'
 
     model_config = SettingsConfigDict(
-        env_file='.env',
+        env_file='../../.env',
```
Comment (mikhaillazko) [inline src/library/database/database_config.py]: bad practice to use relative path, probably you want to use something like: `.env`

File: src/library/minio_aws/minio_config.py
Comment (mikhaillazko) [inline src/library/minio_aws/minio_config.py:1]: minio is just one of the implementation s3 protocol and don't used by AWS, let's rename folder from `minio_aws` to `s3` or `file_storage`

File: src/scrapper/mortgage.py [hunk 2]
```diff
@@ -13,7 +13,24 @@ class MortgageInputData(BaseModel):
...
 
 
-async def fill_mortgage_calc_fields(page: Page, mortgage_data: MortgageInputData) -> Page:
+async def get_website_mortgage_result_table(mortgage_data: MortgageInputData) -> List[Dict[str, any]]:
+    async with async_playwright() as p:
+        url = "https://www.sravni.ru/ipoteka/kalkuljator-ipoteki/"
+        browser = await p.chromium.launch(headless=False, slow_mo=300)
+        context = await browser.new_context(ignore_https_errors=True,
+                                            user_agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+                                                       "(KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36")
```
Comment (mikhaillazko) [inline src/scrapper/mortgage.py]: **nit**: ugly style, as option you can extract this long string to constant

File: src/scrapper/mortgage.py [hunk 3]
```diff
@@ -13,7 +13,24 @@ class MortgageInputData(BaseModel):
...
+        page = await context.new_page()
+        print(f"Goto {url}")
+        await page.goto(url)
+        page = await _fill_mortgage_calc_fields(page, mortgage_data)
+        paymentsDictList = await _get_table_from_page_content(page)
+        await _get_result_page_pdf(page)
+        await context.close()
```
Comment (mikhaillazko) [inline src/scrapper/mortgage.py]: most of the time when you need to call close method i would guess it should support context manager `with` out of the box

File: src/s3/__init__.py
Comment (mikhaillazko) [inline src/s3/__init__.py:1]: looks like can be removed

File: src/scrapper/mortgage.py
Comment (mikhaillazko) [inline src/scrapper/mortgage.py:1]: the better name is `mortgage_parser` or `mortgage_scraper`

File: tests/__init__.py
Comment (mikhaillazko) [inline tests/__init__.py:1]: can be removed

**Review by mikhaillazko — COMMENTED** [review]
i  was expect that schemas will be removed but i still see it in apps

File: src/library/database/models/base_model.py
Comment (Rorzakh-tix) [reply src/library/database/models/base_model.py:1]: done

File: src/library/database/database_config.py (hunk 1 above)
Comment (Rorzakh-tix) [reply src/library/database/database_config.py]: done

File: src/library/minio_aws/minio_config.py
Comment (Rorzakh-tix) [reply src/library/minio_aws/minio_config.py:1]: renamed to s3

File: src/scrapper/mortgage.py (hunk 2 above)
Comment (Rorzakh-tix) [reply src/scrapper/mortgage.py]: extracted to constant

File: src/scrapper/mortgage.py (hunk 3 above)
Comment (Rorzakh-tix) [reply src/scrapper/mortgage.py]: rewrote with context manager

File: src/s3/__init__.py
Comment (Rorzakh-tix) [reply src/s3/__init__.py:1]: done

File: src/scrapper/mortgage.py
Comment (Rorzakh-tix) [reply src/scrapper/mortgage.py:1]: renamed to mortgage_scrapper

File: tests/__init__.py
Comment (Rorzakh-tix) [reply tests/__init__.py:1]: done

**Comment by Rorzakh-tix** [comment]
> i was expect that schemas will be removed but i still see it in apps

removed mortgage_schemes

File: src/main.py [hunk 4]
```diff
@@ -24,7 +24,7 @@ async def lifespan(app: FastAPI):
 app.include_router(fastapi_users_ep.get_reset_password_router(), prefix="/auth", tags=["auth"])
 app.include_router(fastapi_users_ep.get_verify_router(UserRead), prefix="/auth", tags=["auth"])
 app.include_router(fastapi_users_ep.get_users_router(UserRead, UserUpdate), prefix="/auth", tags=["auth"])
-app.include_router(router=mortgage_router,prefix="/scrap", tags=["scraping"])
+app.include_router(router=mortgage_router, prefix="/scrap", tags=["scraping"])
```
Comment (mikhaillazko) [inline src/main.py]: nit: `/scrap` -> `/scrapper`

File: src/scrapper/mortgage_endpoints.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+from scrapper.mortgage_model import Mortgage
+from users.user_model import User
+from src.scrapper.mortgage_scrapper import MortgageInputData, get_website_mortgage_result_table
+from src.users.user_manager import current_active_user
+
+mortgage_router = APIRouter(
+)
+
+
+@mortgage_router.post("/")
```
Comment (mikhaillazko) [inline src/scrapper/mortgage_endpoints.py]: please add separate route `/scrap` or `/parse`

File: src/conftests.py [hunk 6]
```diff
@@ -21,8 +21,8 @@
 async_session_maker_test = async_sessionmaker(engine_test, class_=AsyncSession, expire_on_commit=False)
 
 
-@pytest.fixture(scope="function")
-async def prepare_database():
+@pytest.fixture(scope="session")
```
Comment (mikhaillazko) [inline src/conftests.py]: it should have scope=function if we name it as session in DB terms

Comment (mikhaillazko) [reply src/conftests.py]: looks like you want to have two separate fixture one for setup database when tests running and clean up when all test complete and another fixture to work with database in each test case 
```
def setup_database(): ... - have session scope
def session(setup_database): .... - have function scope
```

**Review by mikhaillazko — APPROVED** [review]

**Comment by mikhaillazko** [comment]
also would better provide more human readable name for pull request and fill description with detail bullet list what was done

File: src/main.py (hunk 4 above)
Comment (Rorzakh-tix) [reply src/main.py]: changed the prefix /scrap to /scrapper in mortgage_router

File: src/scrapper/mortgage_endpoints.py (hunk 5 above)
Comment (Rorzakh-tix) [reply src/scrapper/mortgage_endpoints.py]: added separate route "/scrap" instead of "/" for mortgage_router

File: src/conftests.py (hunk 6 above)
Comment (Rorzakh-tix) [reply src/conftests.py]: Moved "setup_database()" fixture into scope=session. Add "session" fixture with scope=function. Removed unnecessary functions and constants. Removed print function

**Pull request merged**